packages/sdk/src/entities/InlineTool.ts

```diff
@@ -51,7 +78,13 @@ export interface InlineTool extends Omit<InlineToolVersion2, 'save' | 'checkStat
   /**
    * Method for creating wrapper element of the tool
    */
-  createWrapper(): HTMLElement;
+  createWrapper(data?: InlineToolFormatData): HTMLElement;
+
+  /**
+   * Create element for toolbar, which will form data required for inline tool
+   * @param callback - callback function that should be triggered, when data is formed, to apply format to model
+   */
+  createDataFormElement?(callback: (data: InlineToolFormatData) => void): DataFormElementWithOptions | null;
```

Comment:
it is not always a form. In v2 it is called "actions". Lets preserve v2 interface where it is called `renderActions()`